models/articles.model.js

```diff
@@ -7,4 +7,27 @@ exports.fetchArticleById = (article_id) => {
         console.log(result)
```

Comment:
Whoopps... This looks like a leftover `console.log` - remember we should treat submitting a PR as a signal that this code is completely finished and considered ready for production. 
console.logs should be removed regularly! 
(I will forgive a `console.log` that is in a 500 Internal Server error handler :smile: )